CPS-Sticky-Stake/README.md

```diff
@@ -0,0 +1,132 @@
+---
+CPS: ????
+Title: Lost Stake
+Category: ?
+Status: Open
+Authors:
+    - Ryan Wiley <rian222@gmail.com>
+    - Rich Manderino [ECP] 
+    - W Cataldo 
+    - Carlos Lopez de Lara 
```

Comment:
SPOs and other community advocates that have been following this & similar issues for years will naturally know who/what ECP (@EarnCoinPool) is, but considering the general audience and proposed duration of this problem / message we will need something more generally recognisable at least during the review period & submission of whatever CIPs you think may follow in the near term.

Also @CarlosLopezDeLara could use a credential about his current work arrangements.  It is understood from the context that CIP submissions don't necessary reflect the views of the companies that the authors are working for at the time... and the open-source declaration is thankfully enough to pre-empt any issues of IP ownership. 😎